.travis.yml

```diff
@@ -4,7 +4,7 @@ python:
   - "3.7"
   - "3.8"
 env:
-  - DEP=[all]
+  - DEP=[all, doc, dev]
```

Comment:
does this start three jobs? or just one with everything? shouldn't this just be "all" then?